### PR TITLE
Major: Add Twitter Integration & Switch Database

### PR DESCRIPTION
**Pull request by human058382928**

### CODE REVIEW

File: .env.example
Comment (whoabuddy) [inline .env.example:1]: I like the approach here of being able to enable/disable each service.

File: lib/db.py
Comment (whoabuddy) [inline lib/db.py:1]: Do we need this or can we use aibtcdev/aibtcdev-services#8 and aibtcdev/aibtcdev-services#19

**Review by whoabuddy — APPROVED** [review]
I see lots of typing cleanup across the tools it'd be great to list what all this PR covers before it goes through. I like the refactor and simplification though, and if we keep the modular design it'll be easy to figure out where to best add things like front-end, back-end, and services.

File: lib/db.py
Comment (human058382928) [reply lib/db.py:1]: This i will delete once i do migration

**Pull request closed without merging**